src/Nox.ClientApi.Tests/Tests/Application/Dto/CountryQualityOfLifeIndexDtoValidatorTests.cs

```diff
@@ -0,0 +1,68 @@
+﻿using FluentAssertions;
+using Xunit.Abstractions;
+using ClientApi.Application.Dto;
+using AutoFixture;
+using Nox.Types;
+
+namespace ClientApi.Tests.Application.Dto
+{
+    [Collection("Sequential")]
+    public class CountryQualityOfLifeIndexDtoValidatorTests : NoxWebApiTestBase
+    {
+        public CountryQualityOfLifeIndexDtoValidatorTests(ITestOutputHelper testOutput,
+            NoxTestContainerService containerService)
+            : base(testOutput, containerService)
+        {
+        }
+
+        [Fact]
+        public void CountryQualityOfLifeIndexDto_RequiredFields_ShouldValidate()
+        {
+            //Arrange
+            var indexDto = new CountryQualityOfLifeIndexDto();
+            
+            //Act
+            var validationResult = indexDto.Validate();
+
+            //Assert
+            validationResult.Should().NotBeNull();
+            validationResult.Should().ContainKey(nameof(indexDto.CountryId));
+        }
+
+        [Fact]
+        public void CountryQualityOfLifeIndexDto_InvalidFields_ShouldValidate()
+        {
+            //Arrange
+            var indexDto = new CountryQualityOfLifeIndexDto()
+            {
+                IndexRating = 0
+            };
+
+            //Act
+            var validationResult = indexDto.Validate();
+
+            //Assert
+            validationResult.Should().NotBeNull();
+            validationResult.Should().ContainKey(nameof(indexDto.CountryId));
+            validationResult.Should().ContainKey(nameof(indexDto.IndexRating));
+        }
+
+        [Fact(Skip = "System.InvalidOperationException : AutoNumber can only be created with FromDatabase.")]
```

Comment:
@rochar @ILoveMyTortoise Skip should be removed once the Key Relationships are fixes